aspnetcore/razor-pages/ui-class.md

```diff
@@ -22,13 +22,13 @@ Razor views, pages, controllers, page models, [Razor components](xref:blazor/cla
 
 # [Visual Studio](#tab/visual-studio)
 
-* From the Visual Studio **File** menu, select **New** > **Project**.
-* Select **ASP.NET Core Web Application**.
-* Name the library (for example, "RazorClassLib") > **OK**. To avoid a file name collision with the generated view library, ensure the library name doesn't end in `.Views`.
-* Verify **ASP.NET Core 3.0** or later is selected.
-* Select **Razor Class Library** > **OK**.
+* From the Visual Studio **File** menu, select **New**, then select **Project**.
+* Select **Razor Class Library**, then select **Next**.
+* Name the library (for example, "RazorClassLib"), then select **Create**. To avoid a file name collision with the generated view library, ensure the library name doesn't end in `.Views`.
+* Verify **ASP.NET Core 3.0** or later is selected. The **Razor Class Library** should remain selected.
```

Comment:
Delete this entire line.  2.1 and higher is supported. This is not a 3.0 version - right?

Reply:
The moniker range in the original doc specified >= 3.0 for this section.  So I thought this section was specific to 3.0.  Will remove line.

Reply:
LIne 5 has `monikerRange: '>= aspnetcore-2.1'` so I assumed this was not using monikers. We don't mix metadata monikers with inline monikers. If this article has multiple version, then keep the version specific code. I didn't see that in the review diffs.

Reply:
Thanks, got it.  The topic original (published) had the mixed metadata + inline moniker problem.  I didn't notice the metadata moniker until you pointed it out. I will keep a watch out for that issue.